heron/healthmgr/src/java/org/apache/heron/healthmgr/HealthPolicyConfig.java

```diff
@@ -42,6 +44,15 @@ String getPolicyClass() {
     return (String) configs.get(PolicyConfigKey.HEALTH_POLICY_CLASS.key());
   }
 
+  public ToggleablePolicy.PolicyMode getPolicyMode() {
+    String configKey = PolicyConfigKey.HEALTH_POLICY_MODE.key();
+    if (configs.containsKey(configKey)
+        && "deactivated".equals((String) configs.get(PolicyConfigKey.HEALTH_POLICY_MODE.key()))) {
```

Comment:
create a static string variable for `deactivated`

Reply:
updated